tests/samplers/test_rejection_sampler.py

```diff
@@ -95,10 +93,6 @@ def test_correct_output_format(which_tokens_accepted: str,
     )
 
     expected_bonus_token_ids = bonus_token_ids.clone()
-    # If bonus tokens disabled. Verify they are set to -1.
-    # See https://github.com/vllm-project/vllm/issues/4212
-    if disable_bonus_tokens:
-        expected_bonus_token_ids = expected_bonus_token_ids * 0 - 1
```

Comment:
we should keep this assertion in; otherwise the `disable_bonus_tokens` path is not tested. it's fine to delete that path but we should delete it if that's what we want.

Reply:
Will add it back. We need another PR to clean up the `disable_bonus_tokens` field.